src/Desktop.Analyzers/Core/DesktopAnalyzersResources.resx

```diff
@@ -376,7 +376,7 @@
     <comment>CA3075</comment>
   </data>
   <data name="DoNotUseSetInnerXmlMessage" xml:space="preserve">
-    <value>{0} uses the unsafe setter of InnerXml property of System.Xml.XmlDocument.</value>
+    <value>Uses the unsafe setter of InnerXml property of System.Xml.XmlDocument.</value>
```

Comment:
The tone of the messages are inconsistent - some point at a problem, some are more active an say "uses x' and some are not even complete sentences. 

@lgolding - can you please provide suggestions for the messages? If you want to tackle this as part of an all-up review of messages that you were planning, then we don't need to block Rick.